source/js/buyers-guide/components/product-quiz/product-quiz.jsx

```diff
@@ -132,6 +132,7 @@ class ProductQuiz extends Component {
               </p>
               <p className="text-font-sans tw-mb-0 tw-font-light">
                 {productName}
+                <span className="tw-block tw-text-red-40">(points: {product.points})</span>
```

Comment:
This is added to make QA on review app easier. This line needs to be remove before merging this PR.